packages/navigator/src/electron-browser/electron-navigator-menu-contribution.ts

```diff
@@ -45,7 +46,7 @@ export class ElectronNavigatorMenuContribution implements MenuContribution, Comm
             execute: () => {
                 // workaround for https://github.com/electron/electron/issues/4349:
                 // use electron.remote.shell to open the window in the foreground on Windows
-                const shell = electron.remote?.shell ?? electron.shell;
+                const shell = isWindows ? electronRemote.shell : electron.shell;
```

Comment:
It might be that this issue is already fixed with the newer Electron version. I will see whether we still need to call the remote shell and get back to you. See [here](https://github.com/eclipse-theia/theia/pull/10523#discussion_r781549858) for the original discussion on this.

Reply:
@paul-marechal I just checked and we still need to call `electronRemote.shell` on Windows. Your change works perfectly fine 👍